test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12615,7 +12619,16 @@ def __init__(self, name=None, attrs={}, pbsconf_file=None, diagmap={},
         self.configd = os.path.join(self.pbs_conf['PBS_HOME'], 'mom_priv',
                                     'config.d')
         self.config = {}
-        self.dflt_config = {'$clienthost': self.server.hostname}
+        if self.platform == 'cray' or self.platform == 'craysim':
+            if self.platform == 'cray':
+                alps_client = self.du.which(exe='apbasil')
```

Comment:
The path to apbasil is known and mentioned in our documentation, we should use it first, just in case the wrong apbasil is in the path.  We can default to using the output of which apbasil if the known path doesn't work.  The known path is /opt/cray/alps/default/bin/apbasil